README.md

```diff
@@ -24,3 +27,65 @@ For the documentation of the module in `README.md`, the following guidelines app
 
 For the short module description (`shortdesc:`), the following guidelines apply:
 - The description should start with a phrase like "The glorb module reticulates splines" or "The tree module can be used to plant trees".
+
+### Module Config
+
+**Module config** is used to allow local-users to see & change the behavior of the module on booted system, in order to improve local-user experience.
```

Comment:
Is this separate from the config passed in via a recipe?

Reply:
Yes. Maybe it can be rephrased better to not confuse recipe.yml

Reply:
I use the term "Post-install module config" now, to not confuse users about "recipe.yml". If better term can be used, feel free to let me know.